modin/backends/pandas/query_compiler.py

```diff
@@ -1094,12 +1094,21 @@ def unstack(self, level, fill_value):
         def map_func(df):
             return pandas.DataFrame(df.unstack(level=level, fill_value=fill_value))
 
+        is_correct_len = True
+        if isinstance(self.index, pandas.MultiIndex):
+            expected_len = 1
+            for lvl in self.index.levels:
+                expected_len *= len(lvl)
+            if len(self.index) * len(self.columns) != expected_len * len(self.columns):
+                is_correct_len = False
+
```

Comment:
as I understood `expected_len` contains value of actual length and expected length is `len(self.index) * len(self.columns)`? If it is, maybe we should rename `expected_len -> actual_len`

Reply:
Thanks, fixed